concent_api/conductor/constants.py

```diff
@@ -0,0 +1,7 @@
+from core.constants import MESSAGE_TASK_ID_MAX_LENGTH
+
```

Comment:
Is it going to cause an error when conductor will work as a separate app, without `core`?

Reply:
It's never going to have to work without `core`. The only things that are disabled when an app is not in `CONCENT_FEATURES` are:
- Its urlconfig
- Its Celery tasks

You can still import stuff without problems.